Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -457,12 +457,19 @@ + (IterableAPI *)sharedInstanceWithApiKey:(NSString *)apiKey andUserId:(NSString
 }
 
 // documented in IterableAPI.h
-+(void) getAndTrackDeeplink:(NSURL *)webpageURL callbackBlock:(ITEActionBlock)callbackBlock
++ (void)getAndTrackDeeplink:(NSURL *)webpageURL callbackBlock:(ITEActionBlock)callbackBlock
 {
     IterableDeeplinkManager *deeplinkManager = [IterableDeeplinkManager instance];
     [deeplinkManager getAndTrackDeeplink:webpageURL callbackBlock:callbackBlock];
 }
 
++ (void)resolveApplinkURL:(NSURL *)url callback:(ITBURLCallback)callback;
```

Comment:
resolveAppLink sounds like a function that should automatically handle calling openUrl. 
Do you think we should do that automatically?

We can overload getAndTrackDeeplink to pass back a URL.

Reply:
I think we should let the app handle logic for handling the url opens. This is how we have it in the sample app
```
        IterableAPI.resolve(applinkURL: url) { (resolvedUrl) in
            if let resolvedUrl = resolvedUrl {
                DeeplinkHandler.handle(url: resolvedUrl) // DeeplinkHandler is their class.
            }
        }

```

Reply:
Making the example `DeeplinkHandler` code available either in support docs or a repository would be helpful for clients